xonsh/xonfig.py

```diff
@@ -848,37 +847,44 @@ def TAGLINES():
         "Python-powered, cross-platform, Unix-gazing shell",
         "Tab completion in Alderaan places",
         "This fix was trickier than expected",
-        "The unholy cross of Bash/Python",
     ]
 
 
 # list of strings or tuples (string, align, fill)
 WELCOME_MSG = [
     "",
-    ("{{INTENSE_WHITE}}Welcome to the xonsh shell ({version}){{RESET}}", "^", " "),
+    ("Welcome to the xonsh shell {version}", "^", " "),
     "",
     ("{{INTENSE_RED}}~{{RESET}} {tagline} {{INTENSE_RED}}~{{RESET}}", "^", " "),
     "",
     ("{{INTENSE_BLACK}}", "<", "-"),
     "",
     (
-        "{{INTENSE_BLACK}}Create ~/.xonshrc file manually or use xonfig to suppress the welcome screen",
+        "{{INTENSE_BLACK}}Create ~/.xonshrc file manually or use xonfig to suppress the welcome message",
         "^",
         " ",
     ),
     "",
     "{{INTENSE_BLACK}}Start from commands:",
     "  {{GREEN}}xonfig{{RESET}} web         {{INTENSE_BLACK}}# Run the configuration tool in the browser to create ~/.xonshrc {{RESET}}",
     "  {{GREEN}}xonfig{{RESET}} tutorial    {{INTENSE_BLACK}}# Open the xonsh tutorial in the browser{{RESET}}",
+    "[SHELL_TYPE_WARNING]",
     "",
     ("{{INTENSE_BLACK}}", "<", "-"),
     "",
 ]
 
 
 def print_welcome_screen():
+    shell_type = builtins.__xonsh__.env.get("SHELL_TYPE")
     subst = dict(tagline=random.choice(list(TAGLINES)), version=XONSH_VERSION)
     for elem in WELCOME_MSG:
+        if elem == "[SHELL_TYPE_WARNING]":
+            if shell_type != "prompt_toolkit":
+                print_color(
+                    "\n{INTENSE_BLACK}Install {RESET}prompt-toolkit{INTENSE_BLACK} to get the best prompt experience.{RESET}"
```

Comment:
can this best-experience be replaced with featureful-prompt or something similar.

Reply:
Give an example please )

Reply:
Install other xonsh dependencies with 'xonsh[full]' to get fully featured prompt.

Reply:
This message will not be compatible for non-pip package managers.

Reply:
It is just an example. It could be "Install prompt-toolkit to use featureful prompt and pygments for syntax color support."

Reply:
Ok, I suggest to wait @gforsyth`s thoughts too )

Reply:
How about:

```
You are currently using the readline backend.  For interactive tab-completion, on-the-fly syntax highlighting, and more, install prompt_toolkit by running xpip install -U xonsh[full]
```

Reply:
And I _think_ that `pip` (and other pypi-based installs) is the only installation method that won't bring down `prompt_toolkit` by default